OasisDEX/oasis-borrow
# Gf/move features toggles to db

# [Move Feature Flags To Database](https://app.shortcut.com/oazo-apps/story/5331/move-feature-flags-from-local-storage-to-db)

  
## Changes 👷‍♀️
- Added feature flags to schema
- Added a feature flags get endpoint on the api
- update the useFeatureFlag file to accommodate for database feature flags and local-storage

If you look at the code you will notice that in the event that the api request fails when fetching the feature flags, it will fall back to what ever is in local storage. 
  
## How to test 🧪
- update database and run the app with node env in production mode
- set a feature flag in the database, you can use the local storage dev tools to check that is has populated correctly.
